substrate/frame/support/src/traits/tokens/fungibles/lifetime.rs

```diff
@@ -34,6 +34,37 @@ pub trait Create<AccountId>: Inspect<AccountId> {
 	) -> DispatchResult;
 }
 
+/// Trait for resetting the team configuration of an existing fungible asset.
+pub trait ResetTeam<AccountId>: Inspect<AccountId> {
+	/// Reset the team for the asset with the given `id`.
+	///
+	/// ### Parameters
+	/// - `id`: The identifier of the asset for which the team is being reset.
+	/// - `owner`: The new `owner` account for the asset.
+	/// - `admin`: The new `admin` account for the asset.
+	/// - `issuer`: The new `issuer` account for the asset.
+	/// - `freezer`: The new `freezer` account for the asset.
+	fn reset_team(
+		id: Self::AssetId,
+		owner: AccountId,
+		admin: AccountId,
+		issuer: AccountId,
+		freezer: AccountId,
+	) -> DispatchResult;
+}
+
+/// Trait for refunding the deposit of a target asset account.
+pub trait Refund<AccountId> {
```

Comment:
also doesn't feel like this belongs in `lifetime.rs`

Reply:
`lifetime` sounds general enough to me. docs now say - `Traits for creating, editing and destroying assets`. do we really want more modules here? `roles` for team reset, and one more for refund?

Reply:
There is already a `roles.rs` for Reset. I still don't think the refund really belongs here, but if there's nowhere better can leave it.